runtime/binding-tcp/pom.xml

```diff
@@ -250,6 +249,14 @@
             <includes>
               <include>org.agrona:agrona</include>
               <include>io.aklivity.zilla:engine</include>
+              <include>jakarta.json:jakarta.json-api</include>
+              <include>jakarta.json.bind:jakarta.json.bind-api</include>
+              <include>org.eclipse:yasson</include>
+              <include>org.leadpony.justify</include>
+              <include>org.leadpony.joy:joy-yaml</include>
+              <include>org.leadpony.joy:joy-core</include>
+              <include>org.snakeyaml:snakeyaml-engine</include>
+              <include>com.ibm.icu:icu4j</include>
```

Comment:
Is there any shade plugin configuration to pull transitive dependencies implicitly?
Otherwise this will be brittle as we change the dependencies in engine over time.